### ISSUE TITLE
QR/BAR code fuunctionality

### ISSUE THREAD

**Issue by rajvaibhavdubey** [fossasia/badgeyay#1729]
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Feature Description**
The user can add QR Code/barcode images on the badge.
I think that bottom right corner on the badge would be suitable for the location of image.

Discussion needed @ParthS007 @gabru-md @yashLadha

**Comment by ParthS007** [comment]
Yes, I agree with @yashLadha. Please explain your approach and we can discuss it.

**Comment by rajvaibhavdubey** [comment]
@ParthS007 @yashLadha This npm [package](https://www.npmjs.com/package/qrcode-generator) looks after the addition of data to QR code and places a qr image at the specified location, for pixelization , I looked up and found that the minimum QR code size to scanning distance should be 10:1,
like in our badges of size 4"x3" the QR code can be of size .5"x.5".

**Comment by rajvaibhavdubey** [comment]
@ParthS007 @yashLadha @gabru-md Should I start working on this ?

**Comment by YogeshSharma0201** [comment]
@rajvaibhavdubey are you still working on this? Let me know if i can work on this.
